### DISCUSSION TITLE
Cannot login to Web interface with new password

### DISCUSSION THREAD

**Discussion by HomeBro** [TriliumNext/Trilium#6038] [Ideas]
I have changed my password in Trilium and now I can't login with the new password in the web interface. When I write the new password in the password box and press Login it just returns to a blank password field with no errors. If I login with a wrong password I'm told that the password is incorrect and I should try again!?!

I've tried to restart the Trilium server without any luck and tried different browsers with and without incognito.

Anyone had the same problem?

/Kim

**Comment by meichthys** [comment]
I haven't seen this reported by anyone else yet, and the password change process seems to work fine for me.

You should be able to[ restore your database](https://triliumnext.pikapod.net/share/backup) to a point in time prior to the password change, then log in with your old password. 

If you continue to have issues changing your password, let us know!
